### PR TITLE
modifiche action logger

### PR DESCRIPTION
**Pull request by DMinnai**

### CODE REVIEW

**Comment by entando-service** [comment]
All is Well

**Comment by entando-service** [comment]
All is Well

File: plugins/entando-plugin-jpactionlogger/src/main/java/com/agiletec/plugins/jpactionlogger/apsadmin/actionlogger/package_en.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
 title.actionLogger.management=Action Logger
-title.actionLogger.management.help=TO be inserted
+title.actionLogger.management.help=The Action Logger plugin provides an interceptor that the developer can lace among the stack of all the actions managed via struts2
```
Comment (nicpuddu) [inline plugins/entando-plugin-jpactionlogger/src/main/java/com/agiletec/plugins/jpactionlogger/apsadmin/actionlogger/package_en.properties]: can `lace`?
is it a typo?

**Comment by entando-service** [comment]
All is Well

File: plugins/entando-plugin-jpactionlogger/src/main/java/com/agiletec/plugins/jpactionlogger/apsadmin/actionlogger/package_en.properties (hunk 1 above)
Comment (DMinnai) [inline plugins/entando-plugin-jpactionlogger/src/main/java/com/agiletec/plugins/jpactionlogger/apsadmin/actionlogger/package_en.properties]: lace o place :)

**Pull request merged**